mlflow/models/cli.py

```diff
@@ -155,9 +155,10 @@ def prepare_env(
 @commands.command("build-docker")
 @cli_args.MODEL_URI
 @click.option("--name", "-n", default="mlflow-pyfunc-servable", help="Name to use for built image")
+@cli_args.ENV_MANAGER
 @cli_args.INSTALL_MLFLOW
 @cli_args.ENABLE_MLSERVER
-def build_docker(model_uri, name, install_mlflow, enable_mlserver):
+def build_docker(model_uri, name, env_manager, install_mlflow, enable_mlserver):
```

Comment:
Can we also add the `@cli_args.MLFLOW_HOME` arg to this method?

Reply:
Thanks for the catch, sure!